tutorial/README.md

```diff
@@ -0,0 +1,158 @@
+# GoBadge Tutorial
+
+## What you need
+
+    - GoBadge aka Adafruit PyBadge
+    - Personal computer with Go 1.18 and TinyGo installed, and a serial port.
+
+## Installation
+
+### Go
+
+If somehow you have not installed Go on your computer already, you can download it here:
+
+https://golang.org/dl/
```

Comment:
New download URL is : https://go.dev/dl/

Reply:
done